WRP-17865: Update package dependencies to fix security vulnerabilities (#10)
enactjs/babel-preset-enact: CHANGELOG.md

## [unreleased]

> * Renamed several used babel plugins from `@babel/plugin-proposal` to `@babel/plugin-transform` due to their renaming in `@babel` version `7.22.0`.

Reviewer: ```suggestion
* Updated dependencies and renamed several used babel plugins from `@babel/plugin-proposal` to `@babel/plugin-transform`.
```
